org-node/Cargo.toml

```diff
@@ -23,3 +23,8 @@ tracing-subscriber = "0.2"
 ureq = { version = "2.1.1", features = ["json"] }
 ethers = { version = "0.4" }
 rustc-hex = { version = "2.1" }
+tracing-stackdriver = { git = "https://github.com/adaszko/tracing-stackdriver.git", optional = true }
```

Comment:
Why are we depending on your fork?

Reply:
There's a patch in that fork.  We should pull that repo under the radicle-dev github org but as usual, I don't have the necessary permissions.

Reply:
Can we contribute the patch upstream instead?

Reply:
There are critical issues without replies.  The repo is unmaintained.

Reply:
Ok, I will give you access to radicle-dev then. Also, I'd move this to the top-level Cargo.toml -- see how other git dependencies are handled there.

Reply:
Done.